### ISSUE TITLE
Block limit deboost

### ISSUE THREAD

**Issue by NickJamesBTC** [twitter/the-algorithm#127]
**Is your feature request related to a problem? Please describe.**
support town square discussion forarve groups, eliminate echo chamber mentality or algorithmic gamification

Users blocking 10,000+ people works to eliminate the possibility of diverse and broad discussion for all

**Describe the solution you'd like**
Tiny de-boost in engagement based on # of people blocked. E.g. 100 blocks == 0.001% Downboost/degradation of algo enhancement/10,000 blocked users == 1%. Numbers can be configured to scale more correctly, should be almost immeasurable or unnoticeable for average user and deter people from blocking anybody and everybody that says something they disagree with.

**Describe alternatives you've considered**
Limit the number of blocks a user is allowed to 10,000 per account or year as this is quite unreasonable 

**Additional context**
Naval or Cernovich social media accounts are great examples of echo chambers built to produce a dopamine feedback loop rather then encourage honest and wide discussion

**Comment by Gallium3D** [comment]
cap

**Comment by NickJamesBTC** [comment]
What does it mean??

**Comment by DavidBuchanan314** [comment]
There are more than 10,000 idiots on twitter, it's important to be able to block them.

**Comment by bruce-macdonald** [comment]
I don't think this will fly under the flag of Elon's free speech absolutism. People can use their freedom to not hear/see voices they know do not contribute to the discussion.

**Comment by darkdevildeath** [comment]
I really appreciate your goals. But I believe that the Tweet does not have an adequate structure to form sophisticated debates.
The character limit makes this a suitable system for you to say what you think without going too deep and without diving into a debate about your opinions.
Traditional systems that promote debates often support extensive discussions, such as GitHub and Facebook. While an emerging debate structure feels more like Discord to me, where people come together in voice channels and talk in real time.

**Comment by seifAlbaghdady** [comment]
If you have blocked over 10,000 people, and you consider all of them to be idiots, then the problem may lie with you

**Comment by aloiscochard** [comment]
I understand the motivation of the OP, that being said I think this is not really related to software engineering as @seifAlbaghdady said.

It might be better to look for someone knowledgeable on the subject, there is some very interesting book about how not to hate everyone and stuff like that...

Anyway, here the idea is to discuss concrete reasonable actions that can be implemented.

As far as I'm concerned, trying to block the entire planet is not going to create the echo chamber you are looking for (nothing new under the sun by the way).
